exchanges/gateio/gateio_wrapper.go

```diff
@@ -2506,16 +2511,18 @@ func getClientOrderIDFromText(text string) string {
 }
 
 // getTypeFromTimeInForce returns the order type and if the order is post only
-func getTypeFromTimeInForce(tif string) (orderType order.Type, postOnly bool) {
+// TODO: Add in price param to correctly determine if order is market or limit as a zero value price must be a market order
```

Comment:
Can you please check out this TODO as you can pass in the price field straight away and change function name.